File: app/src/main/java/com/silverknife/meizhi/MyApplication.java
package com.silverknife.meizhi;

import android.app.Application;

import com.silvericekey.skutilslibrary.SKUtilsLibrary;
import com.silvericekey.skutilslibrary.netUtils.HttpUtils;

/**
 * Created by silverknife on 2017/12/13.
 */

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        SKUtilsLibrary.init(this);
        HttpUtils.init("http://football.dooksport.com/");
    }
}

File: app/src/main/java/com/silverknife/meizhi/app/MyApplication.kt
package com.silverknife.meizhi.app

import com.silvericekey.skutilslibrary.SKUtilsLibrary
import com.silvericekey.skutilslibrary.base.BaseApplication
import com.silvericekey.skutilslibrary.utils.HttpUtil

/**
 * Created by silverknife on 2017/12/13.
 */

class MyApplication : BaseApplication() {

    override fun onCreate() {
        super.onCreate()
        SKUtilsLibrary.init(this)
        HttpUtil.init("http://football.dooksport.com/")
    }

}
